src/NCRichText.cc

```diff
@@ -887,6 +888,7 @@ bool NCRichText::PadTOKEN( const wchar_t * sch, const wchar_t *& ech )
 
 	    if      ( value == L"big" )		token = T_IGNORE;
 	    else if ( value == L"pre" )		token = T_PLAIN;
+	    else if ( value == L"br/" )		token = T_BR;
```

Comment:
I look all tags and the only other non-pair tag that I found was <hr>. 
I added the if statement to set token to T_IGNORE in <hr/> case, as it already does in <hr> case.